lib/library_web/router.ex

```diff
@@ -28,16 +28,13 @@ defmodule LibraryWeb.Router do
   scope "/admin", LibraryWeb do
     pipe_through :browser
 
-    get "/", AdminController, :index
-    get "/search", AdminController, :search
     post "/create", AdminController, :create
     get "/delete/:id", AdminController, :delete
   end
```

Comment:
I didn't catch this earlier. Can you change this to use `resources "/" AdminController, only [:create, :delete]`? 
Then we'll end up with `admin_path  POST    /admin           Library.AdminController :create` and `admin_path  DELETE    /admin           Library.AdminController :delete`.

[More on resources](https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4).

Reply:
This makes a lot of sense, could we raise it as a separate issue and work on it in a different PR?

Reply:
Yes I think that's best actually, since there are a few other things we can go over for the router 👍